dotNETStandard/HealthVault.Mobile/cs/Xamarin.Core/ViewModels/MedicationEditViewModel.cs

```diff
@@ -6,38 +6,46 @@
 using System.Windows.Input;
 using HealthVault.Sample.Xamarin.Core.Services;
 using Microsoft.HealthVault.Clients;
+using Microsoft.HealthVault.Connection;
 using Microsoft.HealthVault.ItemTypes;
+using Microsoft.HealthVault.Person;
 using Microsoft.HealthVault.Vocabulary;
 using Xamarin.Forms;
 
 namespace HealthVault.Sample.Xamarin.Core.ViewModels
 {
     public class MedicationEditViewModel : ViewModel
     {
+        private readonly Medication medication;
+        private readonly IHealthVaultConnection connection;
+
         public MedicationEditViewModel(
             Medication medication,
-            IList<VocabularyItem> ingredientChoices,
-            IThingClient thingClient,
-            Guid recordId, 
+            IHealthVaultConnection connection,
```

Comment:
I thought we were passing only the specific interfaces required by the viewmodel as a design pattern (https://en.wikipedia.org/wiki/Law_of_Demeter)?